assets/properties-panel.css

```
@@ -205,17 +207,22 @@
 .bio-properties-panel-group {
   background-color: var(--group-background-color);
   border-bottom: 1px solid var(--group-bottom-border-color);
+  position: relative;
 }
 
 .bio-properties-panel-group-header {
-  position: relative;
   display: flex;
   flex-direction: row;
   align-items: center;
   font-size: var(--text-size-base);
   height: 32px;
   user-select: none;
   justify-content: space-between;
+  position: relative;  /* browsers not supporting sticky */
+  position: -webkit-sticky;  /* for safari */
+  position: sticky;
+  top: 0px;
```

```suggestion
  top: 0;
```

No need to use unit for zero :)

---

:+1: